src/tunnelbridge_cmd.cpp

```diff
@@ -55,6 +58,12 @@ TileIndex _build_tunnel_endtile; ///< The end of a tunnel; as hidden return from
 /** Z position of the bridge sprites relative to bridge height (downwards) */
 static const int BRIDGE_Z_START = 3;
 
+extern CommandCost IsRailStationBridgeAboveOk(TileIndex tile, const StationSpec *statspec, uint8_t layout, TileIndex northern_bridge_end, TileIndex southern_bridge_end, int bridge_height,
+											  BridgeType bridge_type, TransportType bridge_transport_type);
+
+extern CommandCost IsRoadStopBridgeAboveOK(TileIndex tile, const RoadStopSpec *spec, bool drive_through, DiagDirection entrance,
+										   TileIndex northern_bridge_end, TileIndex southern_bridge_end, int bridge_height,
+										   BridgeType bridge_type, TransportType bridge_transport_type);
```

Comment:
Should be in an appropriate header file instead of here.
Too much indentation on the continuation lines.

Reply:
when i remove the `extern`s and move these declarations to `station_cmd.h` - which seemed like the most logical choice, as `station_cmd.cpp` is where their definition is - and add includes of `bridge.h` and `newgrf_roadstop.h` for `BridgeType` and `RoadStopSpec` respectively, i start getting compilation errors in completely different parts of the code. any idea why that's happening and how i should avoid it?